src/Resources/Resources/help/Get-AzDeployment.md

```diff
@@ -12,41 +12,89 @@ Get deployment
 
 ## SYNTAX
 
-### GetByDeploymentName (Default)
+### SubscriptionWithDeploymentName (Default)
 ```
-Get-AzDeployment [[-Name] <String>] [-ApiVersion <String>] [-Pre] [-DefaultProfile <IAzureContextContainer>]
- [<CommonParameters>]
+Get-AzDeployment -ScopeType <DeploymentScopeType> [[-Name] <String>] [-ApiVersion <String>] [-Pre]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceGroupWithDeploymentName
+```
+Get-AzDeployment -ScopeType <DeploymentScopeType> [-ResourceGroupName] <String> [[-Name] <String>]
+ [-ApiVersion <String>] [-Pre] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ManagementGroupWithDeploymentName
+```
+Get-AzDeployment -ScopeType <DeploymentScopeType> [-ManagementGroupId] <String> [[-Name] <String>]
+ [-ApiVersion <String>] [-Pre] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### TenantWithDeploymentName
+```
+Get-AzDeployment -ScopeType <DeploymentScopeType> [[-Name] <String>] [-ApiVersion <String>] [-Pre]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
 ```
 
 ### GetByDeploymentId
 ```
-Get-AzDeployment [-Id <String>] [-ApiVersion <String>] [-Pre] [-DefaultProfile <IAzureContextContainer>]
+Get-AzDeployment -Id <String> [-ApiVersion <String>] [-Pre] [-DefaultProfile <IAzureContextContainer>]
  [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Get-AzDeployment** cmdlet gets the deployments at the current subscription scope.
-Specify the *Name* or *Id* parameter to filter the results.
+The **Get-AzDeployment** cmdlet gets the deployments at a resource group, subscription, management group, or tenant scope.
+Specify the *ScopeType* parameter for the scope to get deployments. Specify the *Name* or *Id* parameter to filter the results.
 By default, **Get-AzDeployment** gets all deployments at the current subscription scope.
 
 ## EXAMPLES
 
 ### Example 1: Get all deployments at subscription scope
 ```
-PS C:\>Get-AzDeployment
+PS C:\>Get-AzDeployment -ScopeType Subscription
 ```
 
 This command gets all deployments at the current subscription scope.
 
 ### Example 2: Get a deployment by name
 ```
-PS C:\>Get-AzDeployment -Name "DeployRoles01"
+PS C:\>Get-AzDeployment -ScopeType Subscription -Name "DeployRoles01"
 ```
 
 This command gets the DeployRoles01 deployment at the current subscription scope.
 You can assign a name to a deployment when you create it by using the **New-AzDeployment** cmdlets.
 If you do not assign a name, the cmdlets provide a default name based on the template that is used to create the deployment.
 
+### Example 3: Get a deployment by deployment ID
+```
+PS C:\>Get-AzDeployment -Id "/subscriptions/sub-01/providers/Microsoft.Resources/deployments/DeployRole01"
```

Comment:
@Tiano2017 since we're adding examples, can we add the corresponding output from running each command in all six of the examples in this file? This will give users an idea of what is returned when the cmdlet is run

Reply:
Not sure about this. The output is very large. Does it fit in the help file.